xds/internal/xdsclient/xdslbregistry/xdslbregistry.go

```diff
@@ -69,8 +69,6 @@ func ConvertToServiceConfig(lbPolicy *v3clusterpb.LoadBalancingPolicy, depth int
 		converter := m[policy.GetTypedExtensionConfig().GetTypedConfig().GetTypeUrl()]
 		// "Any entry not in the above list is unsupported and will be skipped."
 		// - A52
-		// This includes Least Request as well, since grpc-go does not support
-		// the Least Request Load Balancing Policy.
```

Comment:
How did you find this? :)

Reply:
I actually ran into it while reading the code a while ago... Then I figured I'd grep for "least request" in this PR.